core/src/test/resources/function-tests.json

```diff
@@ -1142,6 +1142,31 @@
       "input" : "\"a\"",
       "output": "\"a\""
    },
+   {
+      "query": "uuid(9223372036854775807, 9223372036854775807)",
+      "input" : "null",
+      "output": "\"7fffffff-ffff-1fff-bfff-ffffffffffff\""
+   },
+   {
+      "query": "uuid(-9223372036854775808, -9223372036854775808)",
+      "input" : "null",
+      "output": "\"80000000-0000-1000-8000-000000000000\""
+   },
+   {
+      "query": "uuid(0, 0)",
+      "input" : "null",
+      "output": "\"00000000-0000-1000-8000-000000000000\""
+   },
+   {
+      "query": "uuid(1234567890, 1234567890)",
+      "input" : "null",
+      "output": "\"00000000-4996-102d-8000-0000499602d2\""
+   },
+   {
+      "query": "uuid(null, null)",
+      "input" : "null",
+      "output": "\"00000000-0000-0000-0000-000000000000\""
+   },
```

Comment:
I don't see any tests of the `uuid` function with just one argument? Surely we need that?

Reply:
actually i don't think so there is a use-case. just added it to cover the case when only one parameter is passed...changed it, so now an exception is raised for that case (only 0 or 2 are allowed)

Reply:
I think that's probably the best solution. 👍